### DISCUSSION TITLE
Quasar QUploader

### DISCUSSION THREAD

**Discussion by ibrahim-developper** [quasarframework/quasar#10496] [CLI - SSR mode]
how i can set the q-uploader display the images horizontally 
thank you.

**Comment by ontwikkelfabriek** [comment]
@ibrahim-developper 

There's no easy setting/prop on the q-uploader to do this. I can think of 2 ways to archive a horizontal display:

1) Quasar is styled with the BEX css methodology, so you could try to write css( use chrome dev tools).
2) Q-upload has the `list` slot, with this slot you can completely **customize** how the images are displayed:
https://quasar.dev/vue-components/uploader#example--custom-files-list
